images/tempest-container/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM centos/centos:stream9
 USER root
 RUN dnf install -y gcc git jq make python3 python3-pip python-setuptools python-requests
-RUN git clone https://git.openstack.org/openstack/tripleo-repos && \
-    pushd tripleo-repos && \
+RUN git clone https://github.com/openstack-k8s-operators/repo-setup && \
+    pushd repo-setup && \
     python3 setup.py install && \
     popd && \
-    /usr/local/bin/tripleo-repos current-tripleo-dev
+    /usr/local/bin/repo-setup current-podified-dev
```

Comment:
some packages are missing from master components, can we move to current-podified antelope like was recently made on install_yamls[1]?

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/8c040b22b2c07ea2a08cdbc9eac73bfed16f6038/devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup.yaml#L26

Reply:
I've updated this accordignly.